hi-ml-cpath/src/health_cpath/utils/tiff_conversion_config.py

```diff
@@ -95,12 +95,12 @@ def create_dataset_csv_for_converted_data(self, output_folder: Path) -> None:
         :param output_folder: The folder where the new dataset csv file will be saved.
         """
         new_dataset_df = deepcopy(self.slides_dataset.dataset_df)
-        new_dataset_df[self.slides_dataset.IMAGE_COLUMN] = (
-            new_dataset_df[self.slides_dataset.IMAGE_COLUMN]
+        new_dataset_df[self.slides_dataset.image_column] = (
+            new_dataset_df[self.slides_dataset.image_column]
             .str.replace(AMPERSAND, self.replace_ampersand_by)
             .map(lambda x: str(Path(x).with_suffix(TIFF_EXTENSION)))
         )
-        new_dataset_path = output_folder / (self.converted_dataset_csv or self.slides_dataset.DEFAULT_CSV_FILENAME)
+        new_dataset_path = output_folder / (self.converted_dataset_csv or self.slides_dataset.dataset_csv.name)
```

Comment:
why not self.slides_dataset.default_csv_filename is it kept as an attribute?

Reply:
Yup.